components/Token/Header.tsx

```diff
@@ -32,6 +32,7 @@ export type Props = {
   currencies: SaleDetailProps['currencies']
   creator: TokenAssetProps['creator']
   owners: TokenAssetProps['owners']
+  ownerCount: TokenAssetProps['ownerCount']
```

Comment:
Have "owner" as singular bring some confusion.
Is it the number of this asset owned by this owner? Or is it the total number of owners for this asset?
Could name it to something like:
```suggestion
  ownersCount: TokenAssetProps['ownersCount']
```
Or even
```suggestion
  numberOfOwners: TokenAssetProps['numberOfOwners']
```
This comment apply everywhere `ownerCount` is used.

Reply:
renamed everywhere to `numberOfOwners`